bower.json

```diff
@@ -34,7 +34,8 @@
     "mapbox.js": "~3.0.1",
     "tableExport.jquery.plugin": "~1.5.0",
     "raven-js": "~3.12.0",
-    "bootstrap-datepicker": "~1.7.0"
+    "bootstrap-datepicker": "~1.7.0",
+    "throbber": "0.0.2"
```

Comment:
I think the name of the package is actually "throbber.js".  The line should probably read: ```"throbber.js": "~0.0.2"```.